web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument-001.html

```diff
@@ -319,6 +333,39 @@
 }, 'Reading from a custom iterator that returns a non-object keyframe'
    + ' should throw');
 
+// The following two tests are currently against spec, but instead match what
+// we intend the spec to be; see https://github.com/w3c/csswg-drafts/issues/2533
```

Comment:
Specifically, this comment is no longer needed since that issue has been addressed.

Reply:
Done